geospaas_processing/copiers.py

```diff
@@ -16,12 +17,13 @@
 class Copier():
     """Copier for datasets"""
 
-    def __init__(self, type_in_flag_file, destination_path, flag_file_request=False,
+    def __init__(self, type_in_flag_file, destination_path, obsoleteness, flag_file_request=False,
                  link_request=False, **criteria):
         self._type_in_flag_file = type_in_flag_file
         self._destination_path = destination_path
         self._flag_file_request = flag_file_request
         self._link_request = link_request
+        self.obsoleteness = obsoleteness
```

Comment:
Why not just put this "obsoleteness" as a parameter to the `delete()` method?

Reply:
Because of OOP paradigm, the obsoleteness is sth that is related to the object(it belongs to the object), not a parameters that can be carried and varied between different functions of an object. SO it is needed to be defined as an attribute of the very object. It will be easier to debug in the future if necessary.